compiler/control/CompileMethod.cpp

```diff
@@ -379,7 +379,7 @@ compileMethodFromDetails(
          // not ready yet...
          //OMR::MethodMetaDataPOD *metaData = fe.createMethodMetaData(&compiler);
 
-         startPC = compiler.cg()->getCodeStart();
+         startPC = (uint8_t*)compiler.getMethodSymbol()->getMethodAddress();
```

Comment:
This was the main change in thinking here. See the commit body (ab6d6a497983599844cce11f5c1c870e9b160d8c) for a full description of why we made this change.

Reply:
To be explicit this is a design decision which I'd like @mstoodle and @aviansie-ben to comment on, and perhaps @Leonardo2718 as well. It is unfortunate we don't have a public API, because in that case `getCodeStart` would likely be a private API internal to the compiler, and we would only provide one public API to extract the "pointer to a function" we compiled with OMR.

Reply:
This approach seems fine to me. 

I agree that having a single public API to extract the function pointer would be ideal. But I don't think that's a problem that needs to be solved here. The approach you've taken is reasonable given what we currently have, IMHO.